CodeGen/kubernetes/helm-chart/cpu-values.yaml

```diff
@@ -0,0 +1,9 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+image:
+  repository: opea/codegen
+  tag: "latest"
+
```

Comment:
Add `resources.request.cpu` value (e.g. 1) so pod get Pod QoS `Burstable` instead of `BestEffort` . This improves performance.